src/search_algo.hpp

```diff
@@ -475,7 +534,9 @@ searchHalfExactImpl(LocalDataHolder<TGlobalHolder> & lH, TSeed && seed)
     lH.cursor_tmp_buffer.clear();
 }
 
-template <typename TGlobalHolder>
+
+template <DbIndexType   c_indexType,
+          typename      TGlobalHolder>
```

Comment:
indexType is already in the globalholder, shouldn't be an extra template arg

Reply:
True!